packages/terra-navigation-layout/src/NavigationLayoutContent.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Redirect,
+} from 'react-router-dom';
+import classNames from 'classnames/bind';
+
+import AppDelegate from 'terra-app-delegate';
+import RoutingStack from './routing/RoutingStack';
+import { supportedComponentBreakpoints, routeConfigPropType } from './configurationPropTypes';
+
+import styles from './NavigationLayoutContent.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The AppDelegate instance that will be propagated to the RoutingStack and its contents.
+   */
+  app: AppDelegate.propType,
+  /**
+   * The routing configuration from which Routes will be generated.
+   */
+  routeConfig: routeConfigPropType,
+  /**
+   * The current responsive size.
+   */
+  navigationLayoutSize: PropTypes.oneOf(supportedComponentBreakpoints),
+  /**
+   * The String path to redirect to should the routes specified by the routeConfig all fail to match.
+   */
+  redirect: PropTypes.string,
+  /**
+   * Flag to enable navigation within the RoutingStack.
+   */
+  stackNavigationIsEnabled: PropTypes.bool,
```

Comment:
Most of our other props start the is at the beginning (isStackNavigationEnabled). I think this reads easier, as the is immediately indicates the prop is a boolean

Reply:
Yeah this is something I go back and forth on. A lot of times when we prefix a prop with `is`, we are describing the Component itself:
<Menu isOpen>
<Input isActive>
<Component is{Adjective}>

In this case, the prop is describing an aspect of the component:
<Component {Aspect}Is{Adjective}>

I feel like this is still in line with the previous usage. If we switch it up to isStackNavigationEnabled, I feel like the prop starts feeling like a question instead of a statement. It feels a little off, but I understand the consistency concerns. Are there better ways to name this prop?